templates/common/on-prem/files/keepalived.yaml

```diff
@@ -166,14 +173,17 @@ contents:
           - name: IS_BOOTSTRAP
             value: "no"
         command:
-        - dynkeepalived
-        - "/var/lib/kubelet/kubeconfig"
-        - "/config/keepalived.conf.tmpl"
-        - "/etc/keepalived/keepalived.conf"
-        - "--api-vips"
-        - "{{- range $index, $ip := onPremPlatformAPIServerInternalIPs . }}{{ if gt $index 0 }},{{end}}{{$ip}}{{end}}"
-        - "--ingress-vips"
-        - "{{- range $index, $ip := onPremPlatformIngressIPs . }}{{ if gt $index 0 }},{{end}}{{$ip}}{{end}}"
+        - /bin/bash
+        - -c
+        - |
+          #/bin/bash
+          # in remote worker case sleep forever
+          if [ -f "/run/nodeip-configuration/notMatchesVips" ]; then
+              sleep infinity
+          fi
+          api_vips={{- range $index, $ip := onPremPlatformAPIServerInternalIPs . }}{{ if gt $index 0 }},{{end}}{{$ip}}{{end}}
```

Comment:
move the rest of the code to else block

Reply:
i would prefer just exit after "sleep infinity" , will be more readable i think

Reply:
Sure